### PR TITLE
Completed string-manipulation-1.

### PR DESCRIPTION
**Pull request by alexdenbaugh**
string-manipulation-1

### CODE REVIEW

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  var lowerCaseCheck = word.toLowerCase();
+  if (lowerCaseCheck === word) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-lower-cased.js:8]: For this portion, you can just do this: `return lowerCaseCheck === word`

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  var upperCaseCheck = word.toUpperCase();
+  if (upperCaseCheck === word) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-upper-cased.js:8]: For this portion, you can just do this: `return upperCaseCheck === word`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! I have a couple improvements you can make here:

**Comment by alexdenbaugh** [comment]
string-manipulation-1-revised

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**